commons-exo-extension/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "meeds-gamification-github",
```

Comment:
shouldn't it be exo-gamification-github ?

Reply:
done